content/docs/2.8/scalers/pulsar.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Apache Pulsar"
+availability = "v2.8"
+maintainer = "Community"
+description = "Scale applications based on an Apache Pulsar topic subscription."
+layout = "scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `pulsar` trigger for an Apache Pulsar topic.
+
+```yaml
+triggers:
+- type: pulsar
+  metadata:
+    adminURL: http://localhost:80
+    topic: persistent://public/default/my-topic
+    subscription: sub1
+    msgBacklogThreshold: '5'
+```
+
+**Parameter list:**
+
+- `adminURL` - Stats URL of the admin API for your topic.
+- `topic` - Pulsar topic. format of `persistent://{tenant}/{namespace}/{topicName}`
+- `subscription` - Name of the topic subscription
+- `msgBacklogThreshold` - Average target value to trigger scaling actions. (default: 10)
```

Comment:
`activationMsgBacklogThreshold` is missing here, please see other scaler's on the proper wording of the description, for example: https://github.com/kedacore/keda-docs/blob/main/content/docs/2.8/scalers/activemq.md